cip/1.accepted/CIP2017-02-06-Regular-Path-Patterns.adoc

```diff
@@ -0,0 +1,160 @@
+= CIP2017-02-06 Regular Path Patterns
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Tobias Lindaaker <tobias.lindaaker@neotechnology.com>
+
+toc::[]
+
+== Regular Path Patterns
+
+Above and beyond the types of patterns that can be expressed in Cypher using the normal path syntax, Cypher also supports what amounts to regular expressions over paths.
+This functionality is called Regular Path Patterns.
+
+A Regular Path Pattern is defined as:
```

Comment:
For 1 - 4 below, it would be great if these could be supplemented with regex notation